src/main/java/com/capgemini/archaius/spring/ArchaiusPropertyPlaceholderConfigurer.java

```diff
@@ -15,16 +15,26 @@
  */
 package com.capgemini.archaius.spring;
 
-import java.io.IOException;
-import java.util.Map;
-import java.util.Properties;
-
 import com.netflix.config.ConcurrentCompositeConfiguration;
+import com.netflix.config.DynamicStringProperty;
 import org.apache.commons.configuration.ConfigurationConverter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.*;
+import org.springframework.beans.factory.BeanDefinitionStoreException;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.beans.factory.config.BeanDefinitionVisitor;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
 import org.springframework.beans.factory.config.PropertyPlaceholderConfigurer;
 import org.springframework.core.io.Resource;
+import org.springframework.util.ObjectUtils;
+
+import java.io.IOException;
```

Comment:
Could you move these back up to the top? (Its a style thing - I'm guessing these moved down automatically based on your IDE-config.)